Src/Extern/openPMD-api/AMReX_ParticlesOPENPMD.H

```diff
@@ -0,0 +1,79 @@
+#ifndef AMREX_PTL_OPENPMD_API_H
+#define AMREX_PTL_OPENPMD_API_H
+
+/**
+*   This file is included from AMReX_ParticleCounter.H
+*   and belongs to the class ParticleContainer_impl.
+*   The namespace amrex was already declared.
+*
+*   Therefore all the definitions in this file belongs to namespace amrex
+*/
+
+struct AMReX_PtlCounter
+{
+  int m_MPIRank = 0;
+  int m_MPISize = 1;
+
+  unsigned long long m_Total = 0;
+
+  std::vector<unsigned long long> m_ParticleCounterByLevel;
+
+  [[nodiscard]] unsigned long long GetTotalNumParticles () const { return m_Total;}
+
+  std::vector<unsigned long long> m_ParticleOffsetAtRank;
+  std::vector<unsigned long long> m_ParticleSizeAtRank;
+};
+
+
+void CountParticles()
+{
+
+  m_PtlCounter.m_MPISize = amrex::ParallelDescriptor::NProcs();
+  m_PtlCounter.m_MPIRank = amrex::ParallelDescriptor::MyProc();
```

Comment:
AMReX includes for `ParallelDescriptor`, `ParallelGather`, `<vector>` missing in this file.

Reply:
This file is currently embedded in AMReX_ParticleContainer.H, which has all the included files declared already.